### ISSUE TITLE
20200819 히든 레이어

### ISSUE THREAD

**Issue by dbgml4647** [dbgml4647/Meachine-Learning-Study#4]
🔥  더 깊은 신경망을 만들기 위해 히든 레이어를 만들어 보자 🔥 

`히든 레이어` - input layer와 output layer 사이에 있는 망
`활성 화 함수 중  swich 함수는 요즘 많이 쓰이는 활성화 함수 , 보통 이걸로 다 써도 댄다 `

X -> H -> Y

`#모델의 구조를 만듭니다`

X= tf.keras.layers.Input(shape=[4])
H = tf.keras.layers.Dense(8, activation='swish')(X)
H = tf.keras.layers.Dense(8, activation='swish')(H)
H = tf.keras.layers.Dense(8, activation='swish')(H)
Y = tf.keras.layers.Dense(3,activation='softmax')(H)
model = tf.keras.models.Model(X,Y)
model.compile(loss='categorical_crossentropy',
              metrics='accuracy')`

**Comment by dbgml4647** [comment]
https://davinci-ai.tistory.com/20

딥러닝 참고 레퍼런스
